<launch>

    <param name="dataroot" type="string" value="/home/lin/code/mmdetection3d/data/nuscenes"/>
    <param name="project_cloud" type="bool" value="false" />
    <param name="frequency" type="int" value="5" />

    <node pkg="nus_pkg" type="nus_pub_box3d.py" name="nus_pub_box3d" output = "screen" />

    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find nus_pkg)/rviz/box3d.rviz" respawn="false" output="screen" /> -->

</launch>